addons/mass_mailing/models/mailing.py

```diff
@@ -227,6 +227,7 @@ def _get_default_mail_server_id(self):
     bounced_ratio = fields.Float(compute="_compute_statistics", string='Bounced Ratio')
     clicks_ratio = fields.Float(compute="_compute_clicks_ratio", string="Number of Clicks")
     next_departure = fields.Datetime(compute="_compute_next_departure", string='Scheduled date')
+    link_trackers_count = fields.Integer(compute="_compute_link_trackers_count", string="Link Trackers Count")
```

Comment:
🧦 : Would put it just above next_departure as it's more related to statistics than next_departure.